src/Uno.UI/UI/Xaml/Controls/Layouter/Layouter.cs

```diff
@@ -130,6 +130,11 @@ public void Arrange(Rect finalRect)
 				}
 
 				ArrangeOverride(finalRect.Size);
+
+				if (_element is FrameworkElement fe)
```

Comment:
`IFrameworkElement`?

Reply:
This interface is public, and it would force the method to become public... maybe with a private interface.